src/Components/Home/Home.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import NewTokenButton from '../NewTokenButton';
+import ExistingTokenButton from '../ExistingTokenButton';
+import './Home.css';
+
+export default function Home({ setToken }) {
+  return (
+    <div className="home">
+      <h1>Welcome to Your Shopping List!</h1>
+      <section>
+        <NewTokenButton setToken={setToken} />
+        <p>- or -</p>
+        <p>Join an existing shopping list by entering a three word token.</p>
+        <ExistingTokenButton setToken={setToken} />
```

Comment:
I really liked how you organized and styled the Home page! This is nitpicky but the naming of the ExistingTokenButton is a little misleading since it is actually a form with a text input and button.